src/interface/src/app/map/map-manager.ts

```diff
@@ -207,10 +207,28 @@ export class MapManager {
     L.geoJSON(area, {
       style: (_) => DRAWING_STYLES,
       pmIgnore: false,
-      onEachFeature: (_, layer) => {
-        layer.addTo(this.drawingLayer);
-        this.addClonedPolygons(layer);
-        layer.on('pm:edit', ({ layer }) => this.editHandler(layer));
+      onEachFeature: (feature, layer) => {
+        if (feature.geometry.type === 'MultiPolygon') {
```

Comment:
Ideally we'll do this just when validating, but there is an issue on the API that is not saving the mutlipolygon areas correctly, so for now we are doing this.